fec/management/commands/createsu.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+import os
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        if not User.objects.filter(username="admin").exists():
+            User.objects.create_superuser("cgarretty", "admin@admin.com",
```

Comment:
I assume the first argument is the username. Should that just be `admin` or something?